package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "coverage": "nyc report --reporter=text-lcov | coveralls",
     "pretest": "standard",
-    "test": "nyc tap --no-cov tests/**/*.js",
+    "test": "nyc tap 'test/{unit,integration}/**/*-test.js'",
```

Comment:
yes, there were not many tests before, now there are. The previous one would have worked too, I only learned eventually that we have to add `'` around the file pattern, otherwise bash will do the globbing instead of tap